### ISSUE TITLE
Unable to resolve "AccessibilityInfo" 

### ISSUE THREAD

**Issue by pranav-manik** [jacob-hyde/react-native-editable-table#3]
receiving the following error using Expo

```
Unable to resolve "AccessibilityInfo" from "node_modules/react-native-editable-table/node_modules/react-native/Libraries/react-native/react-native-implementation.js"
Failed building JavaScript bundle
```
have tried reinstalling all node modules, could this be an issue with Expo integration?
